Sources/StreamChatUI/ChatMessageList/ChatMessageListRouter.swift

```diff
@@ -59,7 +59,7 @@ open class _ChatMessageListRouter<ExtraData: ExtraDataTypes>:
         threadVC.channelController = client.channelController(for: channel.cid)
         threadVC.messageController = client.messageController(
             cid: channel.cid,
-            messageId: message.id
+            messageId: message.parentMessageId ?? message.id
```

Comment:
I don't think we should fallback to `message.id`. Calling this when `message.parentMessageId == nil` is a programmer error.

Reply:
Its not actually a programmer error. This is valid logic, since if theres no parentId it means the message is the root if the thread, so we need to pass its own id. But I see that it is hidden a bit and is not clear. So the best way is to provide a comment or do this a bit differently, maybe pass the id only on the argument 🤔

Reply:
aaah, I got it now! Thanks for the explanation. A comment in the code is enough for this 👍

Reply:
✅